src/Cosi-BLS/new-transactions.lisp

```diff
@@ -1389,35 +1380,42 @@ ADDRESS here is taken to mean the same thing as the public key hash."
 
 (defparameter *wait-for-tx-count-tick-print* ".")
 (defparameter *wait-for-tx-count-tick-every* 10)
-(defparameter *wait-for-tx-count-message* "Waiting for tx count: ")
+(defparameter *wait-for-tx-default-tx-types* '(:spend :spend-cloaked))
 
-(defun wait-for-tx-count (n &key interval timeout node
-                                 tick-print tick-every message)
+(defun spend-tx-types-p (tx-types)
+  (equal '#.(sort (copy-list '(:spend :spend-cloaked)) #'string<)
```

Comment:
Sure this shouldn't be a membership test instead of a list equality test?

Reply:
It tests set-equal on two sets A, B. True if A is (:foo :bar) and B is (:bar foo), thanks to sorting both lists. The first list is pre-sorted, saving a bit of time. The lists are tiny, so it's trivial time. Just didn't want to worry about it.

Reply:
why not just use (NULL (SET-DIFFERENCE a b)) ?